src/tests/java/picard/analysis/CollectWgsMetricsTest.java

```diff
@@ -0,0 +1,171 @@
+package picard.analysis;
+
+import htsjdk.samtools.SAMException;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMFileWriter;
+import htsjdk.samtools.SAMFileWriterFactory;
+import htsjdk.samtools.SAMReadGroupRecord;
+import htsjdk.samtools.SAMRecord;
+import htsjdk.samtools.SAMRecordSetBuilder;
+import htsjdk.samtools.metrics.MetricsFile;
+import htsjdk.variant.utils.SAMSequenceDictionaryExtractor;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import picard.cmdline.CommandLineProgramTest;
+import picard.sam.SortSam;
+
+import java.io.*;
+import java.util.Random;
+
+/**
+ * Tests for methods in CollectWgsMetrics
+ *
+ *
+ * @author Kylee Bergin
+ */
+
+public class CollectWgsMetricsTest extends CommandLineProgramTest {
+
+    private final static File DICT_DIR = new File("testdata/picard/sam/CollectGcBiasMetrics/");
+    private final static File TEST_DIR = new File("testdata/picard/sam/");
+    private final File dict = new File(DICT_DIR, "MSmallHeader.dict");
+    private File tempSamFile;
+    private File outfile;
+    private final static int LENGTH = 99;
+
+    private final static String sample = "TestSample1";
```

Comment:
static constants should be capitalized -- unless Picard conventions say otherwise, of course.